### DISCUSSION TITLE
Song select filters go into dropdown while there's still a lot of usable space

### DISCUSSION THREAD

**Discussion by Jeglerjeg** [ppy/osu#19276] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/31293237/180212127-64db3688-40cf-4849-8fe7-2f04243991fc.png)

**Comment by peppy** [comment]
This whole layout is going to change in the upcoming redesign (probably going to become a dropdown..?)

**Comment by frenzibyte** [comment, marked as answer]
Since it was improvable I've went ahead and PR'd a fix for it (https://github.com/ppy/osu/pull/19295)
